test/event/handlers/GhostPingHandlerTest.ts

```diff
@@ -51,6 +51,22 @@ describe("GhostPingHandler", () => {
 			expect(messageMock.calledOnce).to.be.false;
 		});
 
+		it("does not send a message when it's author is a bot", async () => {
+			const message = discordMock.getMessage();
+			const messageMock = sandbox.stub(message.channel, "send");
+
+			const author = discordMock.getUser();
+			author.bot = true;
+
+			message.author = author;

```

Comment:
This looks weird. I tried just doing `message.author.bot = true` (as I have done in the tests for `MessagePreviewService`) but for some reason `message.author.bot` was evaluating to `false`, making the test fail. Then I tried this and it worked...